Makefile

```diff
@@ -231,6 +232,13 @@ dist/python_stdlib.zip: $(call rwildcard,src/py/*) $(CPYTHONLIB)
 dist/test.html: src/templates/test.html
 	cp $< $@
 
+dist/makesnap.mjs: src/templates/makesnap.mjs
+	cp $< $@
+
+dist/snapshot.bin: all-but-packages dist/pyodide-lock.json dist/makesnap.mjs
+	cd dist && node --experimental-wasm-stack-switching makesnap.mjs
```

Comment:
Question: Is `--experimental-wasm-stack-switching` always needed making a snapshot?

Reply:
No, but the variable for whether or not it is turned on is stored in the snapshot. Since we run `test_stack_switching.py` against node and assume it will be on, we need this for the tests to pass.